### PR TITLE
Play sound on all OS

### PR DESCRIPTION
**Pull request by SamTheisens**
Current sound at end of build is OS (Mac) specific. It will result in an error on Linux or Windows machines
----
📕 [TECH-438](https://vandebron.atlassian.net/browse/TECH-438) Usability issues with MPyL sam@vandebron.nl 
[p1685111170053379](https://vandebron.slack.com/archives/C04KP2UFMMJ/p1685111170053379) 

🏗️ Build [7](https://jenkins.k8s-serv-backend.vdbinfra.nl/job/MPyL%20Pipeline%20-%20Test/job/PR-129/7/display/redirect) ✅ Successful, started by _Daniel Koves_  
🚀  *enrichChargeSessionsJob*, *job*, *kong-sync*, *[nodeservice](https://nodeservice-129.test.nl/)*, *[sbtservice](https://sbtservice-129.test.nl/)*  

[TECH-438]: https://vandebron.atlassian.net/browse/TECH-438?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/mpyl/cli/commands/build/__init__.py [hunk 1]
```diff
@@ -1 +1,21 @@
...
+@dataclass(frozen=True)
+class Sound(Enum):
+    SUCCESS = 'Glass.aiff'
+    FAILURE = 'Sosumi.aiff'
+
+
+def play_sound(sound: Sound):
+    if shutil.which('afplay') is None:
+        Console().bell()
+        return
```
Comment (koeves) [inline src/mpyl/cli/commands/build/__init__.py:19]: Maybe it's more clear to check for the type of platform mpyl is running on?
There's the `platform` package with `platform.system()` and also `os.name`.

**Review by koeves — APPROVED** [review]
Nice to see you Sam 😁 
Minor comment only

**Review by Jorg88 — APPROVED** [review]

**Comment by SamTheisens** [comment]
Finding out if you're running on mac is unfortunately not straightforward https://stackoverflow.com/questions/1854/how-to-identify-which-os-python-is-running-on

Utlimately I think checking the presence of `afplay` is more reliable, because even if you find out which version of mac you're on, it's still not going to work if it doesn't have `afplay`. WDYT?

**Comment by koeves** [comment]
> Finding out if you're running on mac is unfortunately not straightforward https://stackoverflow.com/questions/1854/how-to-identify-which-os-python-is-running-on
> 
> Utlimately I think checking the presence of `afplay` is more reliable, because even if you find out which version of mac you're on, it's still not going to work if it doesn't have `afplay`. WDYT?
> 
> By the way @koeves, I will leave the decision to merge the PR(s) to you all if that's okay? 👍 I don't want to interrupt your workflow and I don't have the same abilities to test against the monorepo for example.

Let's stick with `afplay` for now then!
And sounds good! 👍

**Pull request merged**